cerberus/api.py

```diff
@@ -125,31 +124,35 @@ def complete(self, request, pk=None):
         return self.change_state("complete", request)
 
     @action(detail=True, methods=["PUT"])
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Ensure API endpoint renaming is reflected in the frontend.

The renaming of the 'move_booking' method to 'move' changes the API endpoint. Make sure this change is updated in any frontend code that calls this endpoint to avoid breaking the application.